tools/debug/debugsymbolviewer.py

```diff
@@ -361,7 +372,9 @@ def print_heap_corruption_data(log_file):
 
 	if (addr_type == 1):
 		print_wrong_sp(log_file)
-	elif (addr_type == 2):
+	if (addr_type == 2):
                 print_heap_corruption_data(log_file)
-	else:
+	if (addr_type == 3):
+                print_running_work_function(log_file)
+	if (addr_type == 0):
```

Comment:
Could you let me know why this is changed to `if`, not `elif`? Is it possible to have multiple values at once?

Reply:
Yeah, it could be called multiple times. But `addr_type` can have only one value at one time. Then `if / elif` is correct. Could you check that?

Reply:
Updated